core/src/main/java/de/danoeh/antennapod/core/feed/FeedMedia.java

```diff
@@ -168,10 +167,6 @@ public MediaType getMediaType() {
     }
 
     public void updateFromOther(FeedMedia other) {
-        // reset to new if feed item did link to a file before
-        if(TextUtils.isEmpty(download_url) && !TextUtils.isEmpty(other.download_url)) {
```

Comment:
so, do we never have the case where we have a media item but the download_url is empty?

i.e. why not keep this in?